CHANGELOG.md

```diff
@@ -5,12 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
 ## Unreleased
 
 ### Fixed
 
 - SpotifyException now thrown when a request fails & has no response ( fixes #571, #581 )
 
+## [2.16.0] - 2020-09-18
```

Comment:
You can remove this line as 2.16.0 was already released and your changes are still unreleased